projects/observability/src/pages/explorer/explorer.component.scss

```diff
@@ -3,7 +3,7 @@
 .explorer-container {
   display: flex;
   flex-direction: column;
-  padding: 24px;
+  padding: 24px 24px 0;
```

Comment:
with the appropriate table layout now, the bottom padding looks silly and eats up too much room, as the table itself is padded.